<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-NU0FN7VQ" ignored="0" name="BankingApp Test" tests="7" failures="2" timestamp="29 Sep 2019 03:29:27 GMT" time="54.009" errors="0">
  <testcase name="loginTest" time="1.035" classname="com.inetbanking.testCases.TC_LoginTest_001"/>
  <testcase name="loginDDT" time="7.599" classname="com.inetbanking.testCases.TC_LoginDDT_002"/>
  <testcase name="loginDDT" time="3.916" classname="com.inetbanking.testCases.TC_LoginDDT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetbanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:30)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="loginDDT" time="7.63" classname="com.inetbanking.testCases.TC_LoginDDT_002"/>
  <testcase name="loginDDT" time="3.849" classname="com.inetbanking.testCases.TC_LoginDDT_002">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at com.inetbanking.testCases.TC_LoginDDT_002.loginDDT(TC_LoginDDT_002.java:30)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- loginDDT -->
  <testcase name="loginDDT" time="7.523" classname="com.inetbanking.testCases.TC_LoginDDT_002"/>
  <testcase name="testAddCustomer" time="5.754" classname="com.inetbanking.testCases.TC_AddCustomerTest_003"/>
</testsuite> <!-- BankingApp Test -->
